internal/catalogmetadata/client/client.go

```diff
@@ -95,6 +95,7 @@ func (c *Client) Bundles(ctx context.Context) ([]*catalogmetadata.Bundle, error)
 		}
 
 		allBundles = append(allBundles, bundles...)
+		fmt.Println("DEBUG:", "ALL BUNDLES:", allBundles)
```

Comment:
leftover?

Reply:
Oh, good catch. Absolutely leftover. I found an issue I was debugging as part of this work 😆

Reply:
Removed in [9cecd82](https://github.com/operator-framework/operator-controller/pull/533/commits/9cecd82c89aa82af486c15b987cc97833517ffbf)